ci-operator/step-registry/azure/enable-encryptionathost-day2/azure-enable-encryptionathost-day2-ref.yaml

```diff
@@ -0,0 +1,11 @@
+ref:
+  as: azure-enable-encryptionathost-day2
+  from: upi-installer
```

Comment:
Suggest to use a tag name of upi-installer image.

Reply:
updated